doc/crypto/api/ops/hashes.rst

```diff
@@ -191,6 +191,19 @@ Hash algorithms
     .. note::
         For other scenarios where a hash function based on SHA3 or SHAKE is required, SHA3-512 is recommended. SHA3-512 has the same output size, and a theoretically higher security strength.
 
+.. macro:: PSA_ALG_SHAKE128_256
+    :definition: ((psa_algorithm_t)0x02000015)
```

Comment:
Need to allocate a distinct value for this algorithm id, and not duplicate the SHAKE256-5125 value.